govc/test/library.bats

```diff
@@ -121,7 +121,11 @@ load test_helper
   assert_success
   library_id="$output"
 
-  run govc library.import my-content "$GOVC_IMAGES/$TTYLINUX_NAME.ova"
+  # copy ova to datastore so we can test library.import with an http source
+  run govc datastore.upload "$GOVC_IMAGES/$TTYLINUX_NAME.ova" "$TTYLINUX_NAME.ova"
```

Comment:
Hi @dougm,

I guess I don't understand this part of the test since the use case is for an OVA imported to the CL from an HTTP source. Why is the datastore involved?

Reply:
This just puts the .ova file behind an http endpoint we can use to test `library.import http://...` with.